packages/plugin-ext/src/main/browser/menus/menus-contribution-handler.ts

```diff
@@ -180,8 +190,10 @@ export class MenusContributionPointHandler {
         const command: Command = { id };
         this.commands.registerCommand(command, handler);
 
-        const { group, when } = action;
-        const item: Mutable<TabBarToolbarItem> = { id, command: id, group: group || '', when };
+        const { when } = action;
+        const [group, sort] = (action.group!.trim() || 'navigation').split('@');
```

Comment:
it is still bogus if `action.group` is undefined.

Also, please add a comment how `~~` makes sure that `sort` string is converted to a valid number? It seems to be some smart trick with bitwise operations, but next maintainer will be puzzled about it. Either rewrite it simpler or add a good comment.

Reply:
ok
I add some comment
and test white space
```
' ' => ['navigation', 0]
'1_rest-client@2' => ['1_rest-client', 2]
'navigation@test' => ['navigation', 0]
```